### PR TITLE
chore(*): add examples from deislabs/example-bundles

### PR DESCRIPTION
**Pull request by vdice**
Picks up where https://github.com/cnabio/duffle/pull/877 left off (🙇 @jeremyrickard !)

* Copies over relevant bundles from https://github.com/deislabs/example-bundles to this repo, with intentions of deprecating the former repo
* Adds example bundle build and validation, updates CI

Notes:

* A few excellent example bundles were removed as they relied on behavior in Duffle that no longer exists (namely, mounting the host docker socket).  Therefore, the [example-voting-app-sqlserver](https://github.com/deislabs/example-bundles/tree/master/example-voting-app-sqlserver) and [openfaas](https://github.com/deislabs/example-bundles/tree/master/openfaas) examples have been dropped (originally added in #877).
 * Some others were removed on account of sitting for so long; they'd need a non-trivial amount of work to revitalize.  Ideally, this PR sets us up for future example bundle additions, now that CI vets them.

TODO:
 - [x] test all example bundles

### CODE REVIEW

**Comment by vdice** [comment]
@squillace @carolynvs mentioned in the Feb 19 CNAB meeting that we may wish to add explicit docs/reasoning around these examples.  I think the concern was that we want to make clear that these are examples for how bundles can be constructed via duffle and not so much for bundle consumers.  Do I have that right?  I'd be happy to add docs/amendments to this PR once we settle on messaging.

**Comment by carolynvs** [comment]
Yup, that's right. The audience of the examples are people developing the CNAB spec and duffle (internal). While people may find them useful, they probably are not the best place to start to learn how to make a bundle.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Duffle: The CNAB Installer
-![Build Status](http://badges.technosophos.me/v1/github/build/deislabs/duffle/badge.svg?branch=master)
+![Build Status](http://badges.technosophos.me/v1/github/build/cnab/duffle/badge.svg?branch=master)
```
Comment (radu-matei) [inline README.md]: The badge is still not working here - this should be a separate issue, not blocking this PR.

Comment (vdice) [reply README.md]: Good call; I at least wanted to fix the typo I introduced (`cnab/duffle` -> `cnabio/duffle`) and in the process I updated to a new/running badge server.

File: README.md [hunk 2]
```diff
@@ -105,8 +102,8 @@ Learn more about about CNAB and Duffle, check out our [docs](docs/README.md).
     *Notes:*
     * To build and install bundles, you need access to a Docker engine - it can be Docker for Mac, Docker for Windows, Docker on Linux, or a remote Docker engine. Duffle uses the Docker engine to build the invocation images, as well as for running actions inside invocation images.
     * Duffle has a driver architecture for different ways of executing actions inside invocation images, and more drivers will be available in the future.
-    * Learn more about what a bundle is and its components [here](https://github.com/deislabs/cnab-spec/blob/master/100-CNAB.md).
-    * Get a feel for what CNAB bundles look like by referencing the [bundles repo](https://github.com/deislabs/bundles) on github.
+    * Learn more about what a bundle is and its components [here](https://github.com/cnabio/cnab-spec/blob/master/100-CNAB.md).
+    * Get a feel for what CNAB bundles written for duffle look like by referencing the [examples directory](./examples).
```
Comment (technosophos) [inline README.md]: not: Either Duffle or `duffle`.

**Comment by vdice** [comment]
@radu-matei @carolynvs PTAL when convenient

File: brigade.js [hunk 3]
```diff
@@ -61,15 +70,32 @@ function test() {
   // Run Go unit tests
   job.tasks = [
     `cd ${localPath}`,
-    "make build-all-bins test"
+    "make build-all-bins test",
+    `cp bin/* ${sharedStorage.path}`
```
Comment (carolynvs) [inline brigade.js]: Dang, we have shared storage across jobs now? 💯

File: brigade.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+      "@brigadecore/brigade-utils": "0.4.1"
```
Comment (carolynvs) [inline brigade.json]: This is wonderful ❤️

File: docs/guides/bundle-guide.md [hunk 5]
```diff
@@ -124,5 +124,5 @@ The output of `duffle install` comes from the run script. `hello world` is print
 
 ## Notes and Next steps
 
-- There are alternatives to defining a custom `run` tool. See examples of more complex and different bundles [here](https://github.com/deislabs/bundles).
-- Read more about the CNAB spec in the [docs](https://github.com/deislabs/cnab-spec/blob/master/100-CNAB.md)
+- There are alternatives to defining a custom `run` tool. See examples of more complex and different bundles [here](../../examples).
```
Comment (carolynvs) [inline docs/guides/bundle-guide.md]: Just a tip, you can use absolute paths like `/examples`, which can be easier to deal with sometimes than relative references when the nesting is deep. GitHub knows that it is relative to the repo and will change it to `https://github.com/deislabs/duffle/examples`.

**Pull request merged**